insalan/tournament/models.py

```diff
@@ -548,6 +613,13 @@ class Manager(models.Model):
         blank=True,
         default=None,
     )
+    pseudo = models.CharField(
+        max_length=42,
+        validators=[MinLengthValidator(1)],
+        null=False,
+        verbose_name=_("Pseudo"),
+        default='',
+    )
```

Comment:
If it's still about in-game name, we don't need any for the managers

Reply:
Yes, I just put it for simplicity for the frontend '^^